pom.xml

```diff
@@ -598,12 +611,6 @@
                 <version>${dep.drift.version}</version>
             </dependency>
 
-            <dependency>
-                <groupId>io.airlift.tpch</groupId>
-                <artifactId>tpch</artifactId>
-                <version>0.9</version>
-            </dependency>
-
```

Comment:
do not move

Reply:
https://github.com/prestodb/presto/pull/11166#discussion_r206802469